internal/injector/config/builtin.go

```diff
@@ -21,7 +22,7 @@ var builtIn = configGo{
 				ID:             "built.WithOrchestrion",
 				TracerInternal: true, // This is safe to apply in the tracer itself
 				JoinPoint: join.AllOf(
-					join.ValueDeclaration(join.MustTypeName("bool")),
+					join.ValueDeclaration(typed.MustTypeName("bool")),
```

Comment:
Very, very optional: This could be a global variable or functor in the `typed` package.